scripts/build_sc_machine.sh

```diff
@@ -14,10 +14,22 @@ relative()
   realpath --relative-to="$(pwd)" "$1"
 }
 
-build_force=0
-build_tests=0
-release_mode=0
+function usage() {
+  cat <<USAGE
 
+  Usage: $(basename "$0") [OPTION]...
+
+  Options:
+    -f, --force        full rebuild with the deleting of the $(relative "${BINARY_PATH}") and $(relative "${BUILD_PATH}") folders
+    -t, --tests        build in tests mode
+    -r, --release      build in release mode
+    --add-definition   add new definition into cmake build
```

Comment:
Can we rename this flag to be more intuitive? `--cmake-arg`, for example, because you may also pass flags like `-G Ninja` or `-Wdev` using this feature. Also, a shorthand flag for this would be useful too.